config/config/src/index.ts

```diff
@@ -275,6 +276,7 @@ export async function getConfig (
     'embed-readme': false,
     'registry-supports-time-field': false,
     'virtual-store-dir-max-length': 120,
+    'peers-suffix-max-length': 1000,
```

Comment:
There's a case where someone uses an older version of pnpm and causes the lock file to essentially revert. It looks like there's a reasonable judgement call here not to handle this since it would require a full major rev rather.

Reply:
I think 1000 is a big enough number. If someone has long dep paths, there's a big chance they might be getting an error at the moment.

Reply:
Agree with your take!